nuxeo-admin-console-web/angular-app/src/app/features/home/components/probes-summary/probes-summary.component.ts

```diff
@@ -1,8 +1,57 @@
-import { Component } from "@angular/core";
+import { PROBES, PROBES_LABELS } from "./../../home.constants";
+import { Component, OnDestroy, OnInit } from "@angular/core";
+import { Observable, Subscription } from "rxjs";
+import { Store, select } from "@ngrx/store";
+import { HomeState, ProbesInfo } from "../../store/reducers";
+import * as HomeActions from "../../store/actions";
 
 @Component({
   selector: "probes-summary",
   templateUrl: "./probes-summary.component.html",
   styleUrls: ["./probes-summary.component.scss"],
 })
-export class ProbesSummaryComponent {}
+export class ProbesSummaryComponent implements OnInit, OnDestroy {
+  probesData: ProbesInfo[] = [];
+  fetchProbesSubscription = new Subscription();
+  fetchProbes$: Observable<ProbesInfo[]>;
+  PROBES_LABELS = PROBES_LABELS;
```

Comment:
Not required to reinitialize. Can use PROBES_LABELS directly below

Reply:
cannot use PROBES_LABELS without initialisation, in the html